memory.x

```diff
@@ -0,0 +1,13 @@
+MEMORY {
```

Comment:
This file should only be needed for the cross-compiled examples...you'll notice that there's already a memory.x file there. So you should be able to update this file if needed.